scripts/glossary.coffee

```diff
@@ -0,0 +1,99 @@
+# Description:
+#   Bot Glossary
+#
+
+fuzzy = require 'fuzzy'
+
+module.exports = (robot) ->
+  robot.respond /what is (.*)\??/i, (msg) ->
+    term = msg.match[1]
+    term = term.split("\?")[0].trim() if term.length > 0
+    items = [];
```

Comment:
I'd rather have a git hub file that gets pulled instead of hard coding like this

https://github.com/github/hubot-scripts/blob/ec0b06aa9f2169c8427cdf77f49a0de698c969b1/src/scripts/word-of-the-day.coffee#L36

And if anyone wants to implement better code for this please do so.